app/src/main/java/com/example/simplegithubapp/main/MenuPagerAdapter.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.simplegithubapp.main
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import com.example.simplegithubapp.follow.FollowingFragment
+import com.example.simplegithubapp.project.ProjectsFragment
+import com.example.simplegithubapp.repository.RepositoriesFragment
+import com.example.simplegithubapp.follow.FollowersFragment
+import com.example.simplegithubapp.overview.OverviewFragment
+
+class MenuPagerAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {
+    var overview:Fragment=
+        OverviewFragment()
+    var repositories:Fragment=
+        RepositoriesFragment()
+    var projects:Fragment= ProjectsFragment()
+    var followers:Fragment=
+        FollowersFragment()
+    var following:Fragment=
+        FollowingFragment()
+
+    var mData:ArrayList<Fragment> = arrayListOf(overview,repositories,projects,followers,following)
+
+    override fun getItem(position: Int): Fragment {
+        return mData.get(position)
+    }
+
```

Comment:
` val menu = arrayListOf("Overview", "Repositories", "Projects", "Followers", "Following")`
메인액티비티에 해당 코드가 있는데, 이 코드들에서 사용되는 순서랑 같아보여. 만약에 탭의 위치가 바뀐다면 둘 다 수정을 해주어야겠지? 근데 다른개발자는 이게 다른곳에도 있는지 모르고 어댑터만 바꿨다면? 치명적인 버그가 발생할수있을꺼야. 이런걸 응집도가 떨어진다. 라고 말을해  이런 데이터는 한곳에서만 쓰는게 좀더 유지보수 하기 좋을꺼야!

```suggestion
    var mData:ArrayList<Fragment> = arrayListOf(overview,repositories,projects,followers,following)

    override fun getItem(position: Int): Fragment {
        return mData[position]
    }

```
글구 코틀린에선 저렇게 배열처럼 접근가능해